packages/hoppscotch-common/src/helpers/import-export/export/envToJson.ts

```diff
@@ -0,0 +1,49 @@
+import { Environment } from "@hoppscotch/data"
+import { TeamEnvironment } from "~/helpers/teams/TeamEnvironment"
+
+const getEnvironmentJson = (
+  environment: TeamEnvironment | Environment,
+  environmentIndex?: number | "Global" | null
+) => {
+  if ("environment" in environment) {
+    const { ...newEnvironment } = environment.environment
+    delete newEnvironment.id
+
+    return environment.id !== null
+      ? JSON.stringify(newEnvironment, null, 2)
+      : undefined
+  } else {
+    const { ...newEnvironment } = environment
+    delete newEnvironment.id
+
+    return environmentIndex !== null
+      ? JSON.stringify(newEnvironment, null, 2)
+      : undefined
+  }
+}
+
+export const exportJSON = (
```

Comment:
I'd prefer naming it `exportToJson` / `exportAsJson`. Feel free to ignore it if you feel unnecessary.